test/test_pynisher.py

```diff
@@ -260,6 +267,21 @@ def test_crash_unexpectedly(self):
         self.assertEqual(wrapped_function.exit_status, pynisher.SignalException)
         self.assertEqual(wrapped_function.exitcode, 0)
 
+    @unittest.skipIf(not all_tests, "skipping keyboard interruption test")
+    def test_keyboard_interruption(self):
+        print("Testing keyboard interruption.")
```

Comment:
No need to have a print here, I don't really understand why the other tests had a print.

Reply:
So can I remove all of them (`print` in `test_pynisher.py`)?

Reply:
That's fine by me.

Reply:
Ok, then I will remove all.